Packs/AggregatedScripts/Scripts/GetUserData/GetUserData.py

```diff
@@ -992,6 +1105,44 @@ def main():
         if verbose:
             command_results_list.extend(users_readables)
 
+        #################################
+        ### Running for XDR ###
+        #################################
+        readable_output, outputs = get_core_and_xdr_data(  # type: ignore[assignment]
+            modules=modules,
+            brand_name="Cortex XDR - IR",
+            first_command="xdr-list-risky-users",
+            second_command="xdr-list-users",
+            first_arg_name="user_id",
+            first_arg_values=users_names,
+            additional_fields=additional_fields,
+            list_non_risky_users=list_non_risky_users,
+            email_list=users_emails,
+            outputs_key_field="PaloAltoNetworksXDR",
+        )
+        if readable_output and outputs:
+            users_outputs.extend(outputs)
+            users_readables.extend(readable_output)
+
+        #################################
+        ### Running for Core ###
+        #################################
+        readable_output, outputs = get_core_and_xdr_data(  # type: ignore[assignment]
+            modules=modules,
+            brand_name="Cortex Core - IR",
```

Comment:
i see in the ticket: https://jira-dc.paloaltonetworks.com/browse/CRTX-194707
its mentioned "both commands will use same brand name" what are they referring to?